R/verb-joins.R

```diff
@@ -435,6 +464,43 @@ join_vars <- function(x_names, y_names, type, by, suffix = c(".x", ".y"), call =
   )
 }
 
+join_simple_table_alias <- function(table_names, aliases) {
```

Comment:
Would it be simpler to have arguments `x_alias`, `y_alias`, `x_name`, `y_name`? Or if this exists in preparation for having more than two table names, I think you could implement more generically in this PR.

Reply:
Yes, simpler with these arguments and it also showed that the function name should be changed to `join_two_table_alias()`